source/_integrations/dreo.markdown

```diff
@@ -0,0 +1,114 @@
+---
+title: Dreo
+description: Instructions on how to set up Dreo fans within Home Assistant.
+ha_category:
+  - Fan
+ha_release: 2024.12
```

Comment:
This should be updated when the core PR is merged so that it matches the actual release